src/renderer.ts

```diff
@@ -139,6 +137,29 @@ const listItemStyles = (format: Format): SerializedStyles => {
             }
         `}
     `;
+
+    const mediaStyles = css`
+        &::before {
+            background-color: ${neutral[46]};
+        }
+    `
+
+    const liveblogStyles = css`
+        ${darkModeCss`
+            &::before {
+                background-color: ${neutral[86]};
+            }
+        `}
+    `
+
+    switch (format.design) {
+        case (Design.Media):
+            return [baseStyles, mediaStyles];
```

Comment:
FYI, as an alternative to returning an array, you could use the `css` function to get `SerializedStyles` as the return type:

https://github.com/guardian/apps-rendering/blob/683fec79cb0be6a9755603757bc7b8148a32562f/src/components/figCaption.tsx#L148